assets/components/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -32,7 +32,12 @@ const StripePopUpButton = (props: PropTypes) => {
 
   const stripeClick = () => props.onStripeClick(props.amount, props.email);
 
-  return <button onClick={stripeClick}>Add CC</button>;
+  return (
+    <button
+      className="component-stripe-pop-up-button"
+      onClick={stripeClick}
+    >Contribute with debit/credit card</button>
```

Comment:
This was added in #78, I've just added a class and tweaked the copy here. Also, is this not a special case? It's the shared Stripe component.